ironfish-cli/src/commands/chain/export.ts

```diff
@@ -67,7 +67,7 @@ export default class Export extends IronfishCommand {
 
     for await (const result of stream.contentStream()) {
       results.push(result.block)
-      progress.update(result.block?.seq || 0)
+      progress.increment()
```

Comment:
Unfortunately this doesn't work. I think this needs to be  `stop - (result.block?.seq || 0)`.

The reason is that export will export all blocks at each sequence, including forks. That means you will get multiple blocks at sequence 10, which will increment progress for every block at sequence 10, so the total progress will exceed the `stop` value. You can see that here, https://github.com/iron-fish/ironfish/blob/master/ironfish/src/rpc/routes/chain/exportChain.ts#L75-L79

If we didn't export forks, what you wrote would be fine.

Reply:
Thank you for the explanation.

I have updated PR considering forks. Now progress will be calculated relying on a `start` position and current block sequence.